Changelog entry for v0.14.2 (#3809)
GenericMappingTools/pygmt: doc/changes.md

# Changelog

## Release v0.14.2 (2025/02/15)

[![Digital Object Identifier for PyGMT v0.14.2](https://zenodo.org/badge/DOI/10.5281/zenodo.14868324.svg)](https://doi.org/10.5281/zenodo.14868324)

> This is a patch release fixing a critical bug introduced in PyGMT v0.14.1.
> 
> ### Bugs
> 
> - Fix the bug for passing text strings with numeric values ([#3804](https://github.com/GenericMappingTools/pygmt/pull/3804))

Reviewer: Thanks for handling this patch so fast!

Just trying to make it look a bit more consistent with the entry for the path release v0.14.1.
As we only have this bug fix as change, a section called highlights is maybe a bit strange; but I would use “bug fixes” as heading, as we used this until now.

```suggestion

### Bug Fixes

- **Patch release fixing a critical bug introduced in PyGMT v0.14.1**
- Fix the bug for passing text strings with numeric values ([#3804](https://github.com/GenericMappingTools/pygmt/pull/3804))
```
